demisto_sdk/dev_tools/linter.py

```diff
@@ -392,26 +397,34 @@ def _docker_image_create(self, docker_base_image, requirements):
 
     def _docker_run(self, docker_image):
         workdir = '/devwork'  # this is setup in CONTAINER_SETUP_SCRIPT
-        pylint_files = os.path.basename(self._get_lint_files())
+        lint_files = os.path.basename(self._get_lint_files())
 
         run_params = ['docker', 'create', '-w', workdir,
-                      '-e', 'PYLINT_FILES={}'.format(pylint_files)]
+                      '-e', 'PYLINT_FILES={}'.format(lint_files)]
+        run_params.extend(['-e', f'PS_LINT_FILES={lint_files}'])
         if not self.root:
             run_params.extend(['-u', '{}:4000'.format(os.getuid())])
         if not self.run_args['tests']:
             run_params.extend(['-e', 'PYTEST_SKIP=1'])
+            run_params.extend(['-e', 'PS_TEST_SKIP=1'])
         if not self.run_args['pylint']:
             run_params.extend(['-e', 'PYLINT_SKIP=1'])
+        if not self.run_args['pslint']:
+            run_params.extend(['-e', 'PYLINT_SKIP=1'])
+            run_params.extend(['-e', 'PS_LINT_SKIP=1'])
         run_params.extend(['-e', 'CPU_NUM={}'.format(self.cpu_num)])
         run_params.extend(['-e', 'CI={}'.format(os.getenv("CI", "false"))])
-        run_params.extend([docker_image, 'sh', './{}'.format(self.run_dev_tasks_script_name)])
-        output = run_command(' '.join(run_params))
+        run_script_name = self.run_dev_tasks_script_name if self.script_type == TYPE_PYTHON else self.run_dev_tasks_script_pwsh_name
+        run_script = self.run_dev_tasks_script if self.script_type == TYPE_PYTHON else self.run_dev_tasks_script_pwsh
+        run_params.extend([docker_image, 'sh', './{}'.format(run_script_name)])
+        print_v(f'container create: {run_params}')
+        output = subprocess.check_output(run_params, stderr=subprocess.STDOUT, universal_newlines=True)
         container_id = output.strip()
         try:
-            output = output + '\n' + run_command(' '.join(['docker', 'cp', self.project_dir + '/.', container_id +
-                                                           ':' + workdir]), exit_on_error=False)
-            output = output + '\n' + run_command(' '.join(['docker', 'cp', self.run_dev_tasks_script, container_id +
-                                                           ':' + workdir]), exit_on_error=False)
+            output = output + '\n' + subprocess.check_output(['docker', 'cp', self.project_dir + '/.', container_id +
```

Comment:
what was the problem with the run_command?

Reply:
It doesn't check process exit code. also if any of the params ever have whitespace it is problematic. I kind of think we should remove it altogether and use the standard python api of `subprocess.check_output` but I didn't want to start a big refactor.